### PR TITLE
Add summarized data to dashboard, add charts for payment processor, s…

### PR DESCRIPTION
**Pull request by danielabertranou-eb**
…ales flag and currency

### CODE REVIEW

File: revenue_app/urls.py [hunk 1]
```diff
@@ -11,6 +11,7 @@
     TopOrganizersLatam,
     top_events_json_data,
     top_organizers_json_data,
+    dashboard_summary,
```
Comment (eb-francisco) [inline revenue_app/urls.py]: please order this and put everything in camel case notation

File: revenue_app/views.py [hunk 2]
```diff
@@ -273,6 +279,36 @@ def top_events_json_data(request):
...
+    trx.payment_processor.replace('', 'n/a', regex=True, inplace=True)
+    payment_processor = trx.payment_processor.value_counts()
+    sales_flag = trx.sales_flag.value_counts()
+    currency = trx.currency.value_counts()
+    res = json.dumps({
+        'payment_processor': {
+            'labels': payment_processor.index.to_list(),
+            'data': payment_processor.values.tolist(),
+            'backgroundColor': colors_pp,
+            'borderColor': [color.replace('0.2', '1') for color in colors_pp]
```
Comment (eb-francisco) [inline revenue_app/views.py]: nit comma

File: revenue_app/views.py [hunk 3]
```diff
@@ -273,6 +279,36 @@ def top_events_json_data(request):
...
+        },
+        'sales_flag': {
+            'labels': sales_flag.index.to_list(),
+            'data': sales_flag.values.tolist(),
+            'backgroundColor': colors_sf,
+            'borderColor': [color.replace('0.2', '1') for color in colors_sf]
```
Comment (eb-francisco) [inline revenue_app/views.py]: nit comma

File: revenue_app/views.py [hunk 4]
```diff
@@ -273,6 +279,36 @@ def top_events_json_data(request):
...
+        },
+        'currency': {
+            'labels': currency.index.tolist(),
+            'data': currency.values.tolist(),
+            'backgroundColor': colors_c,
+            'borderColor': [color.replace('0.2', '1') for color in colors_c]
```
Comment (eb-francisco) [inline revenue_app/views.py]: same here

File: revenue_app/urls.py (hunk 1 above)
Comment (facundochaud-eb) [reply revenue_app/urls.py]: the CamelCase ones are Class Based Views, and the snake_case ones are Function Based Views

Comment (eb-francisco) [reply revenue_app/urls.py]: gotcha, thanks, just apply the order in that case

**Pull request closed without merging**